nltk/text.py

```diff
@@ -322,18 +341,37 @@ def __len__(self):
     # Interactive console methods
     #////////////////////////////////////////////////////////////
 
-    def concordance(self, word, width=79, lines=25):
-        """
-        Print a concordance for ``word`` with the specified context window.
+    def concordance(self, word, width=79, lines=None, stdout=True):
+        """Generate a concordance for ``word`` with the specified context window.
         Word matching is not case-sensitive.
+
+        :param word: The target word
+        :type word: str
+        :param width: The width of each line, in characters (default=80)
+        :type width: int
+        :param lines: The number of lines to display (default=25)
+-       :type lines: int
+        :param stdout: print concordance [True] or return list [False]
+        :type stdout: bool
+
+        :return: concordance, either printed to stdout or as a list
+        :rtype: None, list
+
+        :param stdout: print concordance to stdout
+        :type stdout: bool
```

Comment:
This is not a good variable name and it will conflict with `sys.stdout` if user imports using namespaces.